openstack/cbshelf/Chart.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: v2
+name: shelfapp
+description: A Helm chart for Shelf Application
+type: application
+version: 0.1.0
+appVersion: 1.16.0
+dependencies:
+  - name: postgresql
+    repository: file://../../common/postgresql
+    version: 0.3.0
+  - name: pgbackup
+    repository: file://../../common/pgbackup
```

Please pull this chart from Chartmuseum.
```suggestion
    repository: https://charts.eu-de-2.cloud.sap
```
Otherwise this reference will break when we update pgbackup the next time. (This does not apply to the postgresql chart which we serve as path dependency for historical reasons.)